### PR TITLE
example: add new example 'cubic planar'

### PR DESCRIPTION
**Pull request by peppsac**
Shows how to use multiple layers with custom transformations.

Yet another example to exercise various features of iTowns :)

cubic_planar

### CODE REVIEW

File: examples/cubic_planar.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    obj.position.copy(offset);
+    obj.rotation.copy(cubeTransformations[index].rotation);
+    parent.add(obj);
+    obj.updateMatrixWorld(true);
+
+
+    const tileLayer = itowns.createPlanarLayer('planar' + wms + index, extent, { object3d: obj });
+    tileLayer.disableSkirt = true;
+
+    itowns.View.prototype.addLayer.call(view, tileLayer);
```
Comment (autra) [inline examples/cubic_planar.js]: view.addLayer(tileLayer) does not work here?

File: examples/cubic_planar.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+// Since PlanarView doesn't create default controls, we manipulate directly three.js camera
+// Position the camera at south-west corner
+c = new itowns.Coordinates('EPSG:3946', extent.west(), extent.south(), 2000);
+view.camera.camera3D.position.set(3, 2, 3);// copy(c.xyz().multiplyScalar(scale.x * 0.99));
+view.camera.camera3D.updateMatrixWorld(true);
+view.camera.camera3D.lookAt(new itowns.THREE.Vector3(0, 0, 0));
+
+const revolution = {};
```
Comment (autra) [inline examples/cubic_planar.js]: `var`!

File: examples/cubic_planar.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+revolution.update = (dt) => {
+    parent.rotateZ(0.01);
+    parent.rotateY(0.005);
+    parent.updateMatrixWorld();
+    view.notifyChange(true);
+};
+
+// view.addFrameRequester(revolution);
```
Comment (autra) [inline examples/cubic_planar.js]: maybe you can remove `revolution` if it's not used. (or have a play/pause button?)

File: examples/cubic_planar.js (hunk 2 above)
Comment (autra) [reply examples/cubic_planar.js]: (the linter should warn you about this and arrow functions. If it does not, please tell me)

**Review by autra — APPROVED** [review]
Very fun example! Thanks!

**Comment by peppsac** [comment]
Thanks @autra for the comments - and sorry I thought I already had fixed all the ES6 quirks :(

**Pull request merged**